.github/workflows/publish.yml

```diff
@@ -61,7 +61,7 @@ jobs:
         with:
           cargo-cache-key: cargo-publish
           toolchain: test
-          components: semver-checks
+          components: release, semver-checks
```

Comment:
This is my own ignorance -- does this work as expected? I thought you had to put lists in between brackets, ie. `[release, semver-checks]`

Reply:
Actually, we are not using `[` and `]` in other places since it does not work having them. 😅